tests/Earthfile

```diff
@@ -212,6 +212,9 @@ ga-no-qemu-group10:
 ga-no-qemu-group11:
     BUILD --pass-args ./autoskip+test-group3
 
+ga-no-qemu-group12:
```

Comment:
this was created because the autoskip groups needed to be split up to avoid OOM issues on the m1 buildkite instance (and `ga-no-qemu-group8` was also getting a bit big)